package.json

```diff
@@ -73,6 +73,7 @@
 		"@lezer/xml": "^1.0.0",
 		"@monaco-editor/react": "^4.1.1",
 		"@popperjs/core": "^2.11.5",
+		"@pubpub/deposit-utils": "^0.1.10",
```

Comment:
This is the package that provides a JSX pragma for creating statically-typed and validated Datacite (and in the future, Crossref) deposits from JSX syntax. Repo here: https://github.com/pubpub/deposit-utils